File: Modulo06/Heranca+Namespace/Professor.cpp
#include "Professor.h"

namespace poo {

    Professor::Professor(string nome, int CPF, double salario) :
    Pessoa(nome, CPF), salario(salario) {
    }

    double Professor::getSalario() const {
        return salario;
    }

    void Professor::imprime() const {
        Pessoa::imprime();
        cout << "Salário: " << getSalario() << endl;
    }
}

File: Modulo12/Professor-CSharp/Professor.cs
using System;

namespace poo {

    class Professor : Pessoa {

        private double salario;
        public Professor(string nome, int CPF, double salario) : base(nome, CPF) {
            this.salario = salario;
        }
        public virtual double getSalario() {
            return salario;
        }

        public override void imprime() {
            base.imprime();
            Console.WriteLine("Salário: " + this.getSalario());
        }
    }

}